src/api/hooks/products.ts

```diff
@@ -0,0 +1,21 @@
+import { useQuery } from '@tanstack/react-query';
+
+import type { ProductOptionsData } from '@/types';
+
+import { fetchInstance } from '../instance';
+
+const getProductOptionsPath = (productId: string) => `/v1/products/${productId}/options`;
+
+export const getProductOptions = async (productId: string): Promise<ProductOptionsData> => {
+  const response = await fetchInstance.get<ProductOptionsData>(getProductOptionsPath(productId));
```

Comment:
아래처럼만 작성되어도 충분할거 같은데 분리하신 배경이 궁금하네요

```suggestion
  const response = await fetchInstance.get<ProductOptionsData>(`/v1/products/${productId}/options`);
```

Reply:
코드를 훑어보니 product api를 여러 곳에서 호출하는 부분이 없네요. 제가 해당 부분을 불러온 이후는 이후에 api를 다른 곳에서 불러올 수 있지 않을까 하는 부분에서 나눴습니다 !